src/jdk.javadoc/share/classes/jdk/javadoc/doclet/package-info.java

```diff
@@ -79,10 +79,13 @@
  *   </dd>
  *
  *   <dt><a id="included"></a>Included</dt>
- *   <dd>An element is considered to be <em>included</em>, if it is
- *       <em>specified</em> if it contains a <em>specified</em> element,
- *       or it is enclosed in a <em>specified</em> element, and is <em>selected</em>.
- *       Included elements will be documented.
+ *   <dd>An element is considered to be <em>included</em>, if it is <em>selected</em> and any of the following it true:
+ *     <ul>
+ *       <li>the element is <em>specified</em>, or
+ *       <li>the element contains a <em>specified</em> element, or
```

Comment:
for clarification: I presume this is recursive/inductive ... i.e this implies all enclosing elements are specified